tests/Main.hs

```diff
@@ -510,6 +510,12 @@ runClashTest = defaultMain $ clashTestRoot
               , hdlSim=[Vivado]
               , buildTargets=BuildSpecific ["tb" <> show n | n <- [(1::Int)..7]]
               }
+          , runTest "XpmCdcHandshake" $ def
+              { hdlTargets=[VHDL, Verilog]
+              , hdlLoad=[]
+              , hdlSim=[Vivado]
+              , buildTargets=BuildSpecific ["tb" <> show n | n <- [(1::Int)..7]]
```

Comment:
```suggestion
              , hdlLoad=[Vivado]
              , hdlSim=[Vivado]
              , buildTargets=BuildSpecific ["tb" <> show n | n <- [(0::Int)..7]]
```

Reply:
This one actually starts at 1! But I'll renumber the tests. Don't know what got to me when I started at 1, phah.

Reply:
🤣 You got me there! This was to see if I was still paying attention, right?